azure-mgmt-datalake-store/HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 
 Release History
 ===============
+0.2.0 (2017-07-25)
+++++++++++++++++++
+* Create an inheritance structure for GET and LIST ADLS accounts.
```

Comment:
ChangeLog should be customer focus: what is the actual difference for my code. If you change 90% of the code, but it's changing nothing, I don't want a ChangeLog. Opposite, if you change only  a comma, but it's breaking I want why.

Then, the current bullet list is more a tasks list than a ChangeLog :)